prodlisting/prodlisting.html

```diff
@@ -0,0 +1,204 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="prodlisting.css">
+    <link rel="stylesheet" href="../index.css">
+    <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css" integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous"/>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="../clhome.css">
+
+    <style>
+        body{
+        background-image: url('../updatedbg.jpeg');
+    background-size: cover;
+    width:100%;
+    height: 100%;
+        }
+    </style>
+</head>
+<style>
+    .card-btn-add{
+        display: inline;
+        display: flex;
+        flex-direction: row;
+        justify-content: space-around;
+    }
+</style>
```

Comment:
Suggestion: < style > tags must be included in head if you want to use them in html file